src/Headers/XssProtection/Header.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types = 1);
+
+namespace Pd\SecurityHeaders\Headers\XssProtection;
+
+final class Header implements \Pd\SecurityHeaders\Headers\IHeader
+{
+
+	public function getName(): string
+	{
+		return 'X-Xss-Protection';
```

Comment:
Nejspíš jsou názvy hlaviček "case-insensitive", ale `X-Xss` se mi moc nelíbí. Většinou jsou to všude vidět buď takhle `X-XSS-Protection` nebo `x-xss-protection`. Nechceš to opravit na jedno z toho?